_docs/setup/kubernetes/helm-install.md

```diff
@@ -14,15 +14,46 @@ type: markdown
 
 Quick start instructions for the setup and configuration of Istio using the Helm package manager.
 
-* Installation with Helm prior to Istio 0.7 is unstable and not recommended.*
+<img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning" style="width: 32px; display:inline" />
+Installation of Istio prior to version 0.8.0 with Helm is unstable and not recommended.
 
 ## Prerequisites
 
-The following instructions require you have access to Helm **2.7.2 or newer** in your Kubernetes environment or
-alternately the ability to modify RBAC rules required to install Helm.  Additionally Kubernetes **1.7.3 or newer**
-is also required.  Finally this Helm chart **does not** yet implement automatic sidecar injection.
+* Kubernetes **1.7.3 or newer** is required.
+* Helm **2.7.2 or newer** is required.
+* If you want to manage Istio releases with [Tiller](https://github.com/kubernetes/helm#helm-in-a-handbasket),
+the capability to modify RBAC rules is required.
 
-## Deploy with Helm
+## Deploy Istio using Helm
+
+There are two techniques for using Helm to deploy Istio.  The first
+technique is to use `helm template` to render a manifest and use `kubectl`
+to create it.
+
+The second technique uses Helm's Tiller service to manage the lifecycle
+of Istio.
+
+### Render Kubernetes manifest with Helm and deploy with kubectl
+
+This is the most heavily tested method of deploying Istio.  During the
+continuous integration automated testing and release process, the
+`helm` binary in `template` mode is used to render the various manifests
+produced for Istio.
+
+1. Create an `istio-auth.yaml` Kubernetes manifest:
+   ```bash
+   helm template install/kubernetes/helm/istio --name istio --set global.controlPlaneSecurityEnabled=true global.mtls.enabled=true global.rbacEnabled=true prometheus.enabled=true > $HOME/istio-auth.yaml
+   ```
+
```

Comment:
I would prefer to teach people istio.yaml instead giving I don't recommend users to start with istio-auth.yaml when they on board their apps onto istio.

Reply:
makes sense- updated.